modules/apps/portal-workflow/portal-workflow-test/src/testIntegration/resources/com/liferay/portal/workflow/instance/tracker/web/internal/portlet/action/test/dependencies/single-approver-workflow-definition.xml

```diff
@@ -0,0 +1,187 @@
+				<template-language>freemarker</template-language>
+				<notification-type>email</notification-type>
+				<notification-type>user-notification</notification-type>
+				<execution-type>onAssignment</execution-type>
+			</notification>
+		</actions>
+		<assignments>
+			<user />
+		</assignments>
+		<transitions>
+			<transition>
+				<name>resubmit</name>
+				<target>review</target>
+			</transition>
+		</transitions>
+	</task>
+	<task>
+		<name>review</name>
+		<metadata>
+			<![CDATA[
+				{
+					"xy": [
+						168,
+						36
+					]
+				}
+			]]>
+		</metadata>
+		<actions>
+			<notification>
+				<name>Review Notification</name>
+				<template>${userName} sent you a ${entryType} for review in the workflow.</template>
+				<template-language>freemarker</template-language>
+				<notification-type>email</notification-type>
+				<notification-type>user-notification</notification-type>
+				<execution-type>onAssignment</execution-type>
+			</notification>
+			<notification>
+				<name>Review Completion Notification</name>
+				<template>
+					<![CDATA[Your submission was reviewed<#if taskComments?has_content> and the reviewer applied the following ${taskComments}</#if>.]]>
+				</template>
+				<template-language>freemarker</template-language>
+				<notification-type>email</notification-type>
+				<recipients>
+					<user />
+				</recipients>
+				<execution-type>onExit</execution-type>
+			</notification>
+		</actions>
+		<assignments>
+			<roles>
+				<role>
+					<role-type>organization</role-type>
+					<name>Organization Administrator</name>
+				</role>
+				<role>
+					<role-type>organization</role-type>
+					<name>Organization Content Reviewer</name>
+				</role>
+				<role>
+					<role-type>organization</role-type>
+					<name>Organization Owner</name>
+				</role>
+				<role>
+					<role-type>regular</role-type>
+					<name>Administrator</name>
+				</role>
+				<role>
+					<role-type>regular</role-type>
+					<name>Portal Content Reviewer</name>
+				</role>
+				<role>
+					<role-type>site</role-type>
+					<name>Site Administrator</name>
+				</role>
+				<role>
+					<role-type>site</role-type>
+					<name>Site Content Reviewer</name>
+				</role>
+				<role>
+					<role-type>site</role-type>
+					<name>Site Owner</name>
+				</role>
+			</roles>
+		</assignments>
+		<transitions>
+			<transition>
+				<name>approve</name>
+				<target>approved</target>
+			</transition>
+			<transition>
+				<name>reject</name>
+				<target>update</target>
+				<default>false</default>
+			</transition>
+		</transitions>
+	</task>
+	<state>
+		<name>approved</name>
+		<metadata>
+			<![CDATA[
+				{
+					"xy": [
+						380,
+						51
+					]
+				}
+			]]>
+		</metadata>
+		<actions>
+			<action>
+				<name>approve</name>
+				<status>0</status>
+				<execution-type>onEntry</execution-type>
+			</action>
+		</actions>
+	</state>
+</workflow-definition>
```

Comment:
@carolmariaabb currently Brian is asking to define workflow definition as a json instead of an xml, maybe he rejects these changes due to this reason.

Reply:
here it is a very simple workflow definition as a json:

https://github.com/liferay/liferay-portal/blob/92f257241e2e88c13859d421ebf74c2a5ac1bdba/modules/apps/portal-security/portal-security-script-management-test/src/testIntegration/resources/com/liferay/portal/security/script/management/web/internal/portlet/action/test/dependencies/workflow-definition-1.json

Reply:
and how to deploy:

https://github.com/liferay/liferay-portal/blob/6696fe6bfe9b103935763696de36ac581b7aa808/modules/apps/portal-security/portal-security-script-management-test/src/testIntegration/java/com/liferay/portal/security/script/management/web/internal/portlet/action/test/GetGroovyScriptUsesMVCResourceCommandTest.java#L340-L343